jira.go

```diff
@@ -119,3 +122,65 @@ func updateSprint(sprintID int, args map[string]string) jira.Sprint {
 
 	return *responseSprint
 }
+
+// A pagination-aware alternative for SprintService.GetIssuesForSprint.
+//
+// https://developer.atlassian.com/cloud/jira/software/rest/#api-rest-agile-1-0-sprint-sprintId-issue-get
+// Pagination: https://developer.atlassian.com/cloud/jira/software/rest/#introduction
+func getIssuesForSprint(sprintID int) []jira.Issue {
+	apiEndpoint := fmt.Sprintf("rest/agile/1.0/sprint/%d/issue", sprintID)
+
+	issues := []jira.Issue{}
+	resp := struct {
+		Issues     []jira.Issue `json:"issues" structs:"issues"`
+		MaxResults int          `json:"maxResults" structs:"maxResults"`
+		StartAt    int          `json:"startAt" structs:"startAt"`
+		Total      int          `json:"total" structs:"total"`
+	}{}
+
+	pos := 0
+	for {
+		req, err := jiraClient.NewRequest("GET", apiEndpoint, nil)
+		perror(err)
+		req.URL.RawQuery = url.Values(map[string][]string{
+			"startAt": {strconv.Itoa(pos)},
+		}).Encode()
+
+		_, err = jiraClient.Do(req, &resp)
+		perror(err)
+
+		issues = append(issues, resp.Issues...)
+		if len(issues) == resp.Total {
```

Comment:
as the doc says, the Total may be not set, so we can't use this to break the loop. IMO, here we can add another check, to check the return issues is empty.